pkg/kubectl/cmd/convert/convert.go

```diff
@@ -76,8 +76,8 @@ type ConvertOptions struct {
 	genericclioptions.IOStreams
 }
 
-func NewConvertOptions(ioStreams genericclioptions.IOStreams) *ConvertOptions {
```

Comment:
unexporting to satisfy golint doesn't seem like a good trade. see https://groups.google.com/forum/#!topic/kubernetes-sig-cli/l8tmJgozneo